### ISSUE TITLE
Multiple simultaneous downloads not possible

### ISSUE THREAD

**Issue by RobCoenen** [BiglySoftware/BiglyBT#2106]
I've been using BiglyBT for the last couple of years. Since being updated to version 2.7.0.2. I cannot download multiple torrents simultaneously. Changing the "Max simultaneous downloads" settings doesn't do the trick.

I've already tried resetting BiglyBT and reinstalling it. 
My Java installation is up-to-date. I'm running Windows 10 20H2 (19042.985)

**Comment by parg** [comment]
'Max Active' is more important - if you have a lot of seeding downloads then these will prevent incomplete ones from starting.

**Comment by parg** [comment]
(or of course 'Min Simultaneous Downloads'...)

**Comment by RobCoenen** [comment]
Thanks for the response. To give some details, I have two torrents in my library (0 done, 2 downloading).

Max simultaneous downloads, max active torrents and min simultaneous downloads is set to 0 and "Don't start any more torrents when upload/download speed limit is reached" is unchecked.

**Comment by parg** [comment]
What state do the downloading torrents show in the Status column?

How many seeding torrents do you have?

**Comment by RobCoenen** [comment]
They were showing queued. I now removed BiglyBT complety including every leftovers (Program Files etc.) and it seems to be working now. Thanks for the help.

**Comment by gusssssss** [comment]
I have this same issue except I am running BiglyBt on Mac. Upon switching to BiglyBT from Vuze, only one torrent will download at a time and the others are queued (even if I stop all active seeds), whereas in Vuze I could have up to four downloading at once according to the same settings.

**Comment by pixelangel** [comment]
I have this same issue except I am running BiglyBt on Mac. Upon switching to BiglyBT from Vuze, only one torrent will download at a time and the others are queued (even if I stop all active seeds), whereas in Vuze I could have up to four downloading at once according to the same settings.

**Comment by parg** [comment]
What are your settings for 'minimum simultaneous downloads' and 'max active torrents' ?

**Comment by pixelangel** [comment]
Thanks for your fast response. I had minimum simultaneous downloads set to 1 and max active torrents set to 6 (this is the configuration that worked without any issue in Vuze), but after your response I experimented with the settings and found that if I made the value for minimum simultaneous downloads the same as the max simultaneous downloads, which is set at 4, the desired number of torrents would begin to download. The same effect could be achieved by setting max simultaneous downloads to 0 or some much higher value and selecting only four torrents but this is obviously undesirable for the sake of good settings and automation.

I'm unsure as to why this change was necessary as the Queue settings panels looks almost identical to Vuze's, but thank you for your help nevertheless – as a fan of Vuze I'm excited to start using BiglyBT.
